packages/react-native/src/generators/application/lib/normalize-options.ts

```diff
@@ -44,8 +48,35 @@ export async function normalizeOptions(
   const iosProjectRoot = joinPathFragments(appProjectRoot, 'ios');
   const androidProjectRoot = joinPathFragments(appProjectRoot, 'android');
   const rootProject = appProjectRoot === '.';
+
+  let e2eWebServerTarget = 'start';
```

Comment:
@xiongemi Could make sure some of the assumptions for `react-native` projects are correct here?

- default `webServerTarget`
- default `port` the app gets served on